slides/index.html

```diff
@@ -32,49 +44,108 @@
 # Dette kurset
 
 ## 3 timer
+
+Blåtann  
+mbed  
+Hjelpemidler
+
 ## 3 oppgaver
 
+Lag en mbed med blåtann  
```

Comment:
Should this rather be "Lag en mbed som tror den er et ibeacon, eller noe i den duren?"